### DISCUSSION TITLE
faker.js on node gulp cant getting started, SyntaxError: Unexpected token '?'

### DISCUSSION THREAD

**Discussion by aboutjquery** [faker-js/faker#1083] [Q&A]
I am not going use faker.js on task but just for generate date.

Just one line code
`import { faker } from '@faker-js/faker';`

error message
`SyntaxError: Unexpected token '?'`

Please let me know how can i getting started, thanks.

**Comment by Shinigami92** [comment]
- Which node version
- Which gulp version 
- do you know if you use esm or cjs
- It would be really helpful if you could provide a reproduction somehow

**Reply by aboutjquery** [reply]
Node v13.14
Gulp 4
I am using something like this in gulpfile.babel.js so i think i using esm
```
import gulp from 'gulp';
import { faker } from '@faker-js/faker';
import concat from 'gulp-concat';

export default ...
```

faker.js has its minimum requirements node gulp?

It working with the old version before. (npm i faker)
I get the error after i update to the new version. (npm i @faker-js/faker)

Thank you so much.

**Reply by ST-DDT** [reply]
Please update to node 14 or downgrade to faker v6.3.1.

**Reply by aboutjquery** [reply]
Its working after downgrade to faker v6.3.1, thanks.

**Reply by prisis** [reply]
@aboutjquery can you please test 7.3?
